tests/config/functional/setup_cfg/issue_3630/1.result.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Here's it's strange that we have no value set for init-hook

Reply:
Yeah, this should have a value I think. But I'm not 100% sure since this is one of the "pre-processed" options so it might be a bit broken.